frontend/src/fixtures/commonsFixtures.js

```diff
@@ -41,7 +45,9 @@ const commonsFixtures = {
             "milkPrice": 10,
             "degradationRate": .5,
             "showLeaderboard": true,
+            "capacityPerUser": 13,
             "carryingCapacity": 42,
+            "effectiveCapacity": 42,
```

Comment:
If the commons has 50 players, and the capacity per user is 13, then 13*50=650.   Shouldn't the effective capacity be 650 in that scenario?